### PR TITLE
prettify

### PR DESCRIPTION
**Pull request by jetersen**
instead doing #11 let's start "small" with prettier.
eslint will come later.

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
```
error husky@4.0.7: The engine "node" is incompatible with this module. Expected version ">=10". Got "8.17.0"
error Found incompatible module.

```

**Comment by jetersen** [comment]
Hmm need to update pipeline to use newer... VS code runs with node 12... so I should be fine going to 12 which is LTS

**Comment by jetersen** [comment]
Separate PR though!

**Comment by jetersen** [comment]
I'll merge this last and rebase after everything is merged 😅

File: README.md [hunk 1]
```diff
@@ -37,38 +38,38 @@ You can specify a glob pattern as the value for schema.json which is the file na
 
 The following tasks can be done using VSCode:
 
-  a) Auto completion (Ctrl + Space):
-    Auto completes on all commands.
+a) Auto completion (Ctrl + Space):
```
Comment (jetersen) [inline README.md:41]: Not render as written: https://github.com/jenkinsci/jcasc-vscode-extension#user-guide

Perhaps best rewritten but better follow up in a separate PR.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -14,25 +14,24 @@ pool:
...
+    displayName: 'Install Node.js'
 
-- task: NodeTool@0
-  inputs:
-    versionSpec: '12.x'
-  displayName: 'Install Node.js'
+  - bash: |
+      /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+      echo ">>> Started xvfb"
+    displayName: Start xvfb
```
Comment (timja) [inline azure-pipelines.yml:25]: is this getting interpreted as part of the step?
image

**Review by timja — APPROVED** [review]
one q on the azure-pipeline, rest looks fine

File: azure-pipelines.yml (hunk 2 above)
Comment (jetersen) [reply azure-pipelines.yml:25]: CI passed, again I cannot see the pipeline getting "Build not found"

One of the minor rendering issue that lingers on GitHub.

Comment (jetersen) [reply azure-pipelines.yml:25]: This is what it looks like locally:
image

Comment (timja) [reply azure-pipelines.yml:25]: > CI passed, again I cannot see the pipeline getting "Build not found"

If you login with GitHub you should be able to see it, / maybe log out and log back in.

Comment (timja) [reply azure-pipelines.yml:25]: Also the output should be on the check:
https://github.com/jenkinsci/jcasc-vscode-extension/pull/24/checks

Comment (jetersen) [reply azure-pipelines.yml:25]: No dice.
I see no reason to use azure pipeline exclusively?
The guide linked even suggest you simply use vsce npm package to deploy. Nothing prevents using GitHub action than.
https://code.visualstudio.com/api/working-with-extensions/continuous-integration#automated-publishing

**Comment by jetersen** [comment]
ripping the band-aid :)

**Pull request merged**